frontend/packages/knative-plugin/src/topology/components/edges/TrafficLink.tsx

```diff
@@ -1,14 +1,19 @@
 import * as React from 'react';
-import { Edge, observer, EdgeConnectorArrow } from '@patternfly/react-topology';
+import {
+  Edge,
+  observer,
+  EdgeConnectorArrow,
+  WithContextMenuProps,
+} from '@patternfly/react-topology';
 import { BaseEdge } from '@console/dev-console/src/components/topology';
 
 import './TrafficLink.scss';
 
 type TrafficLinkProps = {
   element: Edge;
-};
+} & WithContextMenuProps;
```

Comment:
Changes here aren't necessary if you type BaseEdge correctly.

Reply:
updated